### PR TITLE
Esupgrade 1.7.3

### PR DESCRIPTION
**Pull request by paul-e-cooley**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.sonian</groupId>
   <artifactId>elasticsearch-jetty</artifactId>
-  <version>1.7.3-BV</version>
+  <version>1.7.3-SNAPSHOT</version>
```
Comment (jonaf) [inline pom.xml:9]: Is this what we want, pre-release? (Is there a jenkins job that does the release to bump the actual released version to `1.7.3`?)

Comment (paul-e-cooley) [inline pom.xml:9]: Right now, we've been doing branch builds. I can also play with the release job that's already in jenkins and get that done, but figured we'd get through the CR's before I went ahead and did that.

Comment (mkelkarbv) [inline pom.xml:9]: I think we should release a ES jetty version and have it here as well as polloi-ops.

**Comment by mkelkarbv** [comment]
LGTM apart from comments!

**Pull request merged**